### PR TITLE
(TO) Repository-centric docs

### PR DESCRIPTION
**Pull request by hestonhoffman**
### What does this PR do? What is the motivation?

The main change I'm documenting here is the shift from test services to repositories. But that change necessitated some other changes.
-  Renames/rewrites the "Search and Manage" page to "Repositories". I changed existing links to "search" to point to the explorer docs, which is a better place for users to go if they want to know how to search for tests. This page wasn't particularly useful and didn't really cover "managing" or "searching" in much depth.
- Various fixes to links and instructions relating to the test service/repository changes.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/tests/repositories.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+The [Test Optimization settings][3] page gives you an overview of the features enabled on each of your repos together with any overrides you've applied. Select a repo to configure the following repository capabilities:
+- **[GitHub Comments][4]**: Show summaries of your test results directly in pull requests.
+- **[Auto Test Retries][5]**: Retry failing tests to avoid failing your build due to flaky tests.
+- **[Early Flake Detection][6]**: Identify flaky tests early in the development cycle.
+- **[Test Impact Analysis][7]**: Automatically select and run only the relevant tests for a given commit based on the code being changed.
+
+### Overrides for test services
+
+If you have a suite of test services that require their own configuration, you can override your default repository settings.
```
Comment (KickingTheTV) [inline content/en/tests/repositories.md:65]:
```suggestion
If you have test services that require their own configuration, you can override your default repository settings.
```

File: content/en/tests/branches.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  tag: "Documentation"
+  text: "Learn how to manage flaky tests"
+- link: "/tests/repositories"
+  tag: "Documentation"
+  text: "Gain insight into test performance in your repositories"
+---
+
+## Overview
+
+The [Branches][1] view of the Tests page lists all branches from all {{< tooltip glossary="test service" >}}s that have reported test results. This tab is useful for seeing the status of tests run on your code branches and troubleshooting test failures.
```
Comment (KickingTheTV) [inline content/en/tests/branches.md:23]:
```suggestion
The [Branches][1] view of the Repositories page lists all branches from all repositories and {{< tooltip glossary="test service" >}}s that have reported test results. This tab is useful for seeing the status of tests run on your code branches and troubleshooting test failures.
```

File: content/en/tests/branches.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+To filter the list down to only branches that you have committed to, toggle **My Branches** and enter the email address associated with your GitHub account. You can enter multiple email addresses. Edit your address later by clicking **Edit Authors**.
+
+## Test results
+
+For each branch, you can see the test service, the number of failed, passed, and skipped tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```
Comment (KickingTheTV) [inline content/en/tests/branches.md:29]:
```suggestion
For each branch, you can see the repository, test service, the number of failed, passed, and skipped tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```

Comment (KickingTheTV) [inline content/en/tests/branches.md:29]:
```suggestion
For each branch, you can see the repository, test service, the number of failed, passed, and skipped tests, new flaky tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```

File: content/en/tests/branches.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Click on a branch to explore the test details page, which includes information about the branch's latest commits, flaky tests, test performance, common error types, and all test runs.
```
Comment (KickingTheTV) [inline content/en/tests/branches.md:31]:
```suggestion
Click on a branch to explore the commit details page, which includes information about the failed tests, new flaky tests, test regressions, test performance, related pipeline executions, and all test runs against that commit.
```

File: content/en/tests/branches.md (hunk 3 above)
Comment (KickingTheTV) [inline content/en/tests/branches.md:29]:
```suggestion
For each branch, you can see the repositories test service(s), the number of failed, passed, and skipped tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```

File: content/en/glossary/terms/test_service.md [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
 title: test service
+short_definition: "A test service is a group of tests associated with a project or repo. It contains all the individual tests for your code, optionally organized into test suites, which are like folders for your tests."
 core_product:
   - ci-cd
   - synthetic monitoring
 related_terms:
   - test suite
 ---
-In Datadog Synthetic Monitoring and Test Optimization, a test service is generally a group of tests associated with, for example, a project or repo. It contains all the individual tests for your code, optionally organized into test suites, which are like folders for your tests. For more information, <a href="/tests/search/">see the documentation</a>.
+In Datadog Synthetic Monitoring and Test Optimization, a test service is generally a group of tests associated with, for example, a project or repo. It contains all the individual tests for your code, optionally organized into test suites, which are like folders for your tests. For more information on Test Optimization, <a href="/tests/">see the documentation</a>.
```
Comment (KickingTheTV) [inline content/en/glossary/terms/test_service.md:11]:
```suggestion
In Datadog Test Optimization, a test service is generally a group of tests associated with, for example, a project or repo. It contains all the individual tests for your code, optionally organized into test suites, which are like folders for your tests. For more information on Test Optimization, <a href="/tests/">see the documentation</a>.
```

Test Services don't apply to Synthetics

File: content/en/tests/flaky_test_management/auto_test_retries.md [hunk 6]
```diff
@@ -131,7 +131,7 @@ You can query the retried tests in the [Test Optimization Explorer][2]: they hav
 
 ## Troubleshooting
 
-If you suspect there are any issues with Auto Test Retries, navigate to the [Test Service Settings page][4], look for your test service, and click **Configure**. Disable Auto Test Retries by clicking on the toggle.
+If you suspect there are any issues with Auto Test Retries, navigate to the [Test Optimization Settings page][4], look for your test service, and click **Configure**. Disable Auto Test Retries by clicking on the toggle.
```
Comment (KickingTheTV) [inline content/en/tests/flaky_test_management/auto_test_retries.md:134]:
```suggestion
If you suspect there are any issues with Auto Test Retries, navigate to the [Test Optimization Settings page][4], look for your repository, and click **Configure**. Disable Auto Test Retries by clicking on the toggle.
```

File: content/en/tests/flaky_test_management/early_flake_detection.md [hunk 7]
```diff
@@ -117,7 +117,7 @@ You can use the following facets to query sessions that run Early Flake Detectio
 
 ## Troubleshooting
 
-If you suspect there are issues with Early Flake Detection, navigate to the [Test Service Settings page][7], look for your test service, and click **Configure**. Disable Early Flake Detection by clicking on the toggle.
+If you suspect there are issues with Early Flake Detection, navigate to the [Test Optimization Settings page][7], look for your test service, and click **Configure**. Disable Early Flake Detection by clicking on the toggle.
```
Comment (KickingTheTV) [inline content/en/tests/flaky_test_management/early_flake_detection.md:120]:
```suggestion
If you suspect there are issues with Early Flake Detection, navigate to the [Test Optimization Settings page][7], look for your repository, and click **Configure**. Disable Early Flake Detection by clicking on the toggle.
```

**Review by KickingTheTV — CHANGES_REQUESTED** [review]
Have a few suggestions, but overall LGTM

File: content/en/tests/repositories.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+The [Repositories page][1] displays aggregated health metrics for the default branch of each repository. This view is useful for gaining a high-level overview of test performance across your organization's repositories.
+
+Use the Repositories page to:
+- See the total number of flaky tests in each repo.
+- Track whether tests are becoming more or less unreliable over time.
+- See the test status for the most recent commit in each repository.
+- See how many {{< tooltip glossary="test service" >}}s are associated with each repo.
+
+<!-- vale Datadog.pronouns = NO -->
```
[the remark this answers is not in the document]
Comment (hestonhoffman) [reply content/en/tests/repositories.md:35]: It turns off a linter that would fire on the word "My"

File: content/en/tests/developer_workflows.md [hunk 9]
```diff
@@ -46,31 +46,20 @@ With this information, developers get instant feedback about their tests results
...
+## Enable test summaries
 
 You can enable test summaries in pull requests with the following steps:
 
 1. Install the [GitHub integration][4]:
    1. Navigate to the **Configuration** tab on the [GitHub integration tile][6] and click **+ Create GitHub App**.
-   2. Give the application read and write permissions for pull requests.
-2. Enable test summaries for one or more test services. You can do this from the [Test Service Settings page][3] or from the commit/branch page.
+   1. Give the application read and write permissions for pull requests.
+1. Navigate to the [Test Optimization Settings page][3] and search for the repository you want to enable. You can also search by repository.
```
Comment (hestonhoffman) [inline content/en/tests/developer_workflows.md:56]:
```suggestion
1. Navigate to the [Test Optimization Settings page][3].
```

File: content/en/tests/developer_workflows.md [hunk 10]
```diff
@@ -46,31 +46,20 @@ With this information, developers get instant feedback about their tests results
...
+1. Select the repository.
```
Comment (hestonhoffman) [inline content/en/tests/developer_workflows.md:57]:
```suggestion
1. Select the repository where you want to enable test summaries.
```

**Review by KickingTheTV — APPROVED** [review]
LGTM

File: content/en/tests/repositories.md [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+Use the [Repositories page][1] to gain insights into test trends in your repositories or debug issues by investigating individual commits.
+
+## Test performance and trends
+
+The [Repositories view][1] displays aggregated health metrics for the default branch of each repository. This view is useful for gaining a high-level overview of test performance and trends across your organization's repositories.
+
+Use the Repositories page to:
+- See the total number of flaky tests in each repo.
+- Track whether tests are becoming more or less unreliable over time.
+- See the test status for the most recent commit in each repository.
```
Comment (rtrieu) [inline content/en/tests/repositories.md:33]:
```suggestion
- See the test status for the most recent commit in each repo.
```

**Review by rtrieu — APPROVED** [review]
One suggestion for consistency, but otherwise, great update. Thanks for this PR!

**Pull request merged**